kernel_tuner/observers/register.py

```diff
@@ -0,0 +1,16 @@
+from kernel_tuner.observers.observer import BenchmarkObserver
+
+class RegisterObserver(BenchmarkObserver):
```

Comment:
I like this new observer, but adding it seems outside the scope of this PR which is about flushing the L2 cache.